data/events.toml

```diff
@@ -0,0 +1,33 @@
+#==============================================================
+######################## EVENT FORMAT #########################
+#==============================================================
+
+#[[event]]
+#    dataTime = "Feb 9th, 2022"          (Any data and time format)
+#    title = "Bayesian Modeling in Biotech"          (Event title)
+#    place = "Online"                                (Event place)
+#    description = "In this talk..."                 (Small description of the event)
+#    bannerImgPath = "/images/events/talk1.jpg"      (Event thumnail/banner img path)
+#    speakerName = "Ravin Kumar"                     (Speaker/Author name)
+#    speakerImgPath = "/images/speakers/ravin.jpg"   (Speaker/Author img path, leave blank if unavailable i.e speakerImgPath="")
+#    speakerSocialLink = "www.github.com"            (Author's social link, leave blank if unavailable i.e speakerSocialLink = "")
+#    readMoreURL = "https://www.meetup.com"          (Event post URL i.e Discourse, Meetup etc)
+#    eventOrder = 2                                  (An integer, used to order events in ascending order)
+#    eventType = "upcoming"                          (must be "upcoming" or "past")
+#    featured = true                                 (A boolean, if true, the event will be displayed on home pade NOTE: We only display the "Featured event section" if there are more than three events to display.)
```

Comment:
Here is a note for the same

Reply:
ah im sorry I should have read the documentation